### PR TITLE
translate `Manual installation` in japanese

### PR DESCRIPTION
**Pull request by chansuke**
## Summary

Part of #880

Translate [Manual insallation](https://biomejs.dev/guides/manual-installation/) page in japanese.

## Test Plan

```shell
cd website
pnpm install
pnpm start
```
and checked the page(including link behavior)

### CODE REVIEW

**Review by nissy-dev — COMMENTED** [review]
Thanks! ! left some comments

File: website/src/content/docs/ja/guides/manual-installation.mdx [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+title: 手動インストール
+description: Biome を手動でインストールする
+---
+
+## Biome を Node.js 無しで使用する
+
+もしあなたが Node や npm (あるいは他のパッケージマネージャー)を使用してない場合、Biome のスタンドアロンなCLIバイナリを使用することは良い選択です。
+言い換えれば、Biomeが `package.json` を持つ唯一の理由であるべきではありません。
```
Comment (unvalley) [inline website/src/content/docs/ja/guides/manual-installation.mdx:9]:
```suggestion
もしあなたが Node.js や npm（あるいは他のパッケージマネージャー）を使用してない場合、Biome のスタンドアロンなCLIバイナリを使用できます。
言い換えれば、`package.json` を持たずに Biome を利用できます。
```

File: website/src/content/docs/ja/guides/manual-installation.mdx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+:::note
+すでにnpmや他のパッケージマネージャーを使用している場合は、パッケージマネージャーを使用することがBiomeの[推奨されるインストール方法](/ja/guides/getting-started#installation)です。
+あなたはツールに既に慣れており、インストールや更新が簡単です。
```
Comment (unvalley) [inline website/src/content/docs/ja/guides/manual-installation.mdx:13]:
```suggestion
すでにnpmや他のパッケージマネージャーを使用している場合は、パッケージマネージャーを使用することがBiomeの[推奨されるインストール方法](/ja/guides/getting-started#installation)です。
慣れているツールの利用は、インストールや更新を簡単にします。
```

File: website/src/content/docs/ja/guides/manual-installation.mdx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+```
+
+## システム要件
+
+- Windows (WSL 含む), macOS, または Linux
+- x86_64 または ARM64
+
+## サポートされるプラットフォーム
+
+Biomeを動作させるためには、プラットフォームに適したバイナリを選択する必要があります。次の表が選択の助けになるでしょう。
```
Comment (unvalley) [inline website/src/content/docs/ja/guides/manual-installation.mdx:31]:
```suggestion
Biomeを動作させるためには、プラットフォームに適したバイナリを選択する必要があります。次の表を参考にしてください。
```

File: website/src/content/docs/ja/guides/manual-installation.mdx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+# Windows (x86_64, Powershell)
+Invoke-WebRequest -Uri "https://github.com/biomejs/biome/releases/download/cli%2Fv<version>/biome-win32-x64.exe" -OutFile "biome.exe"
+```
+
+:::note
+`<version>` の項目を、インストールされたBiome のバージョンに置き換えて下さい。
+:::
+
+これで、 `./biome` を実行することでBiome を使用出来ます。
```
Comment (unvalley) [inline website/src/content/docs/ja/guides/manual-installation.mdx:63]:
```suggestion
これで、 `./biome` を実行することでBiome を使用できます。
```

**Review by unvalley — CHANGES_REQUESTED** [review]
Thank you for translation, I left some suggestions:)

**Comment by chansuke** [comment]
@unvalley I appreciate your feedback! I'll check that later🙇

**Comment by chansuke** [comment]
@nissy-dev @unvalley 
Thank you so much for the review :pray:  I've fixed and rebased. Please take a look when you have a time :bow:

File: website/src/content/docs/ja/guides/manual-installation.mdx [hunk 5]
```diff
@@ -0,0 +1,67 @@
+---
+title: 手動インストール
+description: Biome を手動でインストールする
+---
+
+## Biome を Node.js 無しで使用する
```
[the remark this answers is not in the document]
Comment (chansuke) [reply website/src/content/docs/ja/guides/manual-installation.mdx]: I appreciate for your review

**Review by unvalley — APPROVED** [review]

**Pull request merged**